src/stores/room-list/algorithms/Algorithm.ts

```diff
@@ -331,26 +333,13 @@ export class Algorithm extends EventEmitter {
         this.emit(LIST_UPDATED_EVENT);
     }
 
-    // TODO: Remove or use.
-    protected addPossiblyFilteredRoomsToTag(tagId: TagID, added: Room[]): void {
```

Comment:
I guess this is no longer used...?

Reply:
It was never used, and I kept running into the comment.